### DISCUSSION TITLE
Unexpected replies to commands when tgcf is deployed as userbot

### DISCUSSION THREAD

**Discussion by Cryptotion** [aahnik/tgcf#215] [Q&A] [answered]
How to stop sending this auto reply ' For details visit github.com/aahnik/tgcf '.  If anyone use /help in chat it send this. 

> Edit by @aahnik
> This thing is being triggered by any character followed by the command word

**Comment by aahnik** [comment, marked as answer]
**Unexpected triggering has been fixed in release [v0.2.9](https://github.com/aahnik/tgcf/releases/tag/v0.2.9)** 

**[Click here update to the latest version ➥](https://github.com/aahnik/tgcf/discussions/212)**


If you send `/help` to the bot (or `.help` to the user bot), it will send you the help message.
